tests/test_ds9.py

```diff
@@ -36,9 +36,11 @@
 except Exception:
     ds9 = None
 
+FRAME_NUM = 731  # largish, random number, unlikely to be in use by a user
```

Comment:
This is an arbitrary number, not a random number.

Reply:
Lol, I'll fix that once Jenkins passes and then merge without rerunning, I think that's fine, and not worth restarting Jenkins for.